jsonnet/prometheus-operator/prometheus-operator.libsonnet

```diff
@@ -10,6 +10,7 @@ local defaults = {
     requests: { cpu: '', memory: '' },
   },
   enableReloaderProbes: false,
+  goGC: '30',
```

Comment:
As per https://pkg.go.dev/runtime the default is 100. Should we retain the default value here while allowing it to be configurable or is the idea that we try (and push users to) to take advantage of the optimisation?

Reply:
The latter. My observation is that the memory usage increases a lot initially when the operator populates its local cache but it's transient because most of the data is discarded by the transform func and reclaiming more aggressively than the default is beneficial in this case.

Reply:
Yeah makes sense. Feels pretty low risk so all good with me :)